src/components/Subscription/helper.js

```diff
@@ -34,6 +34,12 @@ export function computeFlagLabel(name, t, lang) {
     return `${beginning}${t(`Subscription.included.numeric_flag_unlimited`)}`
   }
 
+  if (name.includes('max-days')) {
```

Comment:
nit: Why include and not a strict equality with flag name ?

Reply:
That's a good point. I was going for a more generic case, but it doesn't make sense